### DISCUSSION TITLE
How do you use the injectCSS parameter to change the background?

### DISCUSSION THREAD

**Discussion by OtiDioti** [GitSquared/edex-ui#1014] [Questions] [answered]
hey,
I am kinda new to all this linux thing, but I do have some programming basis.
I wanted to know how I could set up my own background in the UI. 
I saw in the "theme wiki" there is mention of a injectCSS parameter present in the theme files. I got access to said parameter, but I really do not get how to use it to fix my own background.

Thanks for the help.

**Comment by GitSquared** [comment, marked as answer]
It's used to plug your own CSS styling code into the UI. When edex is opened, you can use ctrl+shift+i to bring up the internal chromium developer tools - use that to explore how the DOM is structured and what styles are applied, then you can override them with injectCSS.

**Reply by OtiDioti** [reply]
I fixed it, changed the background and decreased the overall terminal opacity.
Although now I am stuck again: I injected in my theme the parameter #main_shell_innercontainer{opacity: 0.5}. This has the effect of decreasing the terminal opacity, although I cant seem to be able to separately control the text opacity inside the terminal (which I would like to max out). 
Any suggestions?

**Reply by GitSquared** [reply]
Opacity affects the whole element, you could try setting the background-color and color properties with rgba (transparent) values instead?
